tools/switch_tplg/README.md

```diff
@@ -0,0 +1,12 @@
+## Topology Switch Tool
+
```

Comment:
Please add a prefix to the commit subject like `tools: `. Also add some implementations detail in the commit message. 

Like for example: we remove and then insert back all SOF related modules thus triggering topology reload.

Reply:
@dbaluta @AnneOnciulescu would it make sense to update the sof-insert/sof-remove scripts in the sof-test repo to do what you're suggesting already? And as for choosing a different topology should be just a simple command to se the kernel module params to choose the new topology isnt it? Can you please elaborate the need of the python tool for choosing a different topology?

Reply:
Could be yes, but then the user needs to run 3 scripts instead of selecting the topology from a GUI. On the other hand we can create a wrapper over existing sof-insert / sof-remove scripts. 

Let me talk to @AnneOnciulescu  and get back with a imporved solution.